extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -830,6 +839,11 @@ export class AmpLiveList extends AMP.BaseElement {
   getUpdateTime() {
     return this.updateTime_;
   }
+
+  sendAmpDomUpdateEvent_() {
+    const event = new Event('amp-dom-update');
```

Comment:
not sure, i was following https://github.com/ampproject/amphtml/blob/master/3p/ampcontext-lib.js#L29